modules/test/playwright/tests/asset-categories-admin-web/pages/AssetCategoriesEditPage.ts

```diff
@@ -60,12 +57,42 @@ export class AssetCategoriesEditPage {
 			await valueInput.fill(value);
 		}
 
-		await this.saveButton.click();
+		if (save) {
+			await this.save();
+		}
+	}
+
+	async fillName(name: string) {
+		await this.descriptionField.waitFor();
+		await this.nameInput.fill(name);
+	}
+
+	async goto(action: string, title: string) {
```

Comment:
This goto may need to be renamed to goToAction and moved to AssetCategoriesAdminPage.ts